src/main/java/de/retest/recheck/cli/subcommands/Show.java

```diff
@@ -70,10 +70,10 @@ private void printDiff() throws TestReportFormatException, IOException {
 		final TestReport filteredTestReport = filter.filter( report );
 		final TestReportPrinter printer = new TestReportPrinter( none() );
 
-		logger.info( "\n{}", printer.toString( filteredTestReport ) );
+		logger.info( "\n\n{}\n\n", printer.toString( filteredTestReport ) );
 
 		if ( filteredTestReport.getDifferencesCount() > 0 ) {
-			logger.info( "\nOverall, recheck found {} difference(s) when checking {} element(s).",
+			logger.info( "Overall, recheck found {} difference(s) when checking {} element(s).",
```

Comment:
Maybe use a different log level to draw the user's attention to the fact that some action is required.

```suggestion
			logger.warn( "Overall, recheck found {} difference(s) when checking {} element(s).",
```